### DISCUSSION TITLE
Disable Dependency Lookup Warnings block on the dependency dashboard.

### DISCUSSION THREAD

**Discussion by MojoJojo86** [renovatebot/renovate#22742] [Suggest an Idea]
We have an issue where in multi module projects, it can't find the pom from another module in the same project.
This is fine as we don't care about it but the fact it displays the Dependency Lookup Warning in the dashboard with the Alert emojis causes support requests with people asking us if everything is ok.

Would it be possible to have a suppression notification to just disable this or does something like this already exist?

Thanks :)

**Comment by rarkins** [comment]
Apart from ignoring such dependencies through config, this could find a home in https://docs.renovatebot.com/configuration-options/#suppressnotifications (but doesn't seem to exist)

**Reply by MojoJojo86** [reply]
That sounds exactly where we thought it would be good to put it. 
Is this something that could be done or would it be an open source contribution? :)

**Reply by rarkins** [reply]
PRs welcome. I suggest adding `dependencyLookupWarnings` to the existing list

**Reply by MojoJojo86** [reply]
Don't suppose you have had any issues with something similar to this when trying to run the yarn test?

`TypeError [ERR_UNKNOWN_FILE_EXTENSION]: Unknown file extension ".ts" for /Users/Joseph.Bradshaw/IdeaProjects/renovateOfficial/tools/generate-schema.ts`

**Reply by viceice** [reply]
no, how you're running the tests?

**Reply by dhduvall** [reply]
I'm getting that error on `yarn build`. The prior `yarn install` works fine except for a handful of warnings. This is a freshly cloned copy of `renovate`, both with node & yarn from homebrew on Mac as well as node & yarn installed as per the instructions on local development. I'm a complete noob when it comes to JS/TS, so I'm not sure what other info to provide (and it probably means I don't have anything weird in my home directory that would be relevant).

**Reply by dhduvall** [reply]
For me, at least, downgrading node from 20 to 18 did the trick. Thanks to @viceice for the suggestion.

**Reply by viceice** [reply]
so it seems like a compabillity issue with ts-node and node v20
